cmake/modules/FindUSD.cmake

```diff
@@ -1,5 +1,26 @@
 # Simple module to find USD.
 
+# Note that on Windows with USD <= 0.19.11, USD_LIB_PREFIX should be left at
+# default (or set to empty string), even if PXR_LIB_PREFIX was specified when
+# building core USD, due to a bug.
+
+# On all other platforms / versions, it should match the PXR_LIB_PREFIX used
+# for building USD (and shouldn't need to be touched if PXR_LIB_PREFIX was not
+# used / left at it's default value).
+
+set(USD_LIB_PREFIX ${CMAKE_SHARED_LIBRARY_PREFIX}
+    CACHE STRING "Prefix of USD libraries; generally matches the PXR_LIB_PREFIX used when building core USD")
+
+if (WIN32)
```

Comment:
I don't have a strong preference on this, but utils.cmake defines `IS_WINDOWS` (along with `IS_LINUX` and `IS_MACOSX`). Do we want to use that here instead of `WIN32`?

Reply:
Indeed we are using IS_XXX and will have some cleanup to do. Good catch.

Reply:
I'll amend the PR...

Reply:
Actually... on second thought, this is a Find*.cmake.  In general, it's nice if these can be made as self-contained and portable as possible, as they're frequently copied between projects.  So not relying on externally defined variables is nice.

So my vote would be to leave it as WIN32... though I don't feel strongly enough to fight for this, if you guys strongly prefer IS_WINDOWS.

Reply:
Very valid point @elrond79! I will let @HamedSabri-adsk comment.

Reply:
@elrond79 Yes, it is a valid point. As you mentioned Find*.cmake are self contained and portable so having IS_WINDOWS in this case will break portability.  Leaving it as WIN32 is fine. Thanks for pointing it out.

Now regarding the PXR_LIB_PREFIX, I noticed that George recently mentioned below patch on the group interest which is a little different than this PR and makes it a bit confusing:

https://github.com/PixarAnimationStudios/USD/commit/1ca7422061007fd7b91db7e68620cdd571dd8cb5

I thought it was decided to not remove PXR_LIB_PREFIX option?